synfig-studio/src/gui/states/state_bone.cpp

```diff
@@ -126,17 +140,13 @@ class studio::StateBone_Context : public sigc::trackable
 	Widget_Distance skel_bone_width_dist;
 	Widget_Distance skel_deform_bone_width_dist;
 
-
-	// Layer creation radio group
+	// Layer Type
 	Gtk::Label layer_label;
-	Gtk::RadioButton::Group radiogroup;
-	Gtk::RadioButton radiobutton_skel;
-	Gtk::RadioButton radiobutton_skel_deform;
-	Gtk::Button create_layer;
-
+	Gtk::ToggleButton layer_skel_togglebutton;
+	Gtk::ToggleButton layer_skel_deform_togglebutton;
+	Gtk::HBox layer_types_box;
```

Comment:
Do you plan to replace deprecated `HBox` in another PR?

Reply:
This should be fixed in the last commit 👍